### ISSUE TITLE
FFmpeg & SVT-HEVC - Patch couldn't be applied with 'git am'. Continuing without patching.

### ISSUE THREAD

**Issue by BlackScreen** [jb-alvarado/media-autobuild_suite#1264]
I tested with SVT-HEVC and FFmpeg today - but sadly without success...

```
18:06:32   Running git update for ffmpeg...
18:06:34 ┌ ffmpeg git ............................. [Newer dependencies]
18:06:34 ├ Changing options to comply to nonfree...
CUDA_PATH environment variable not set or directory does not exist.
ffmpeg-libsvthevc-patches.patch
        Patch couldn't be applied with 'git am'. Continuing without patching.
18:06:37 ├ Compiling static FFmpeg...
18:06:38 ├ Running uninstall...
18:06:39 ├ Running configure...
18:11:03 ├ Running make...
18:22:14 ├ Running install...
18:22:25 ├ Stripping...
18:22:25 └ ffmpeg-git ........................................ [Updated]
```

**Comment by wiiaboo** [comment]
Fixed in b2778b1

**Comment by BlackScreen** [comment]
Working. Thanks!
